client/reader/following/intro.jsx

```diff
@@ -34,19 +31,17 @@ class FollowingIntro extends React.Component {
 			// This is incorrect, but keeping the name for consistency.
 			// This event is more like `calypso_reader_following_rendered_for_new_reader`.
 			recordTrack( 'calypso_reader_following_intro_render' );
-		}
-		if ( props.isNewReader === true && ! isEven( props.userId ) ) {
-			// Added _actual for when a new Reader and odd userid sees the intro card
+
+			// Added _actual because the original event was fired incorrectly during A/B test
 			recordTrack( 'calypso_reader_following_intro_render_actual' );
```

Comment:
Let's remove the `calypso_reader_following_intro_render_actual` but keep the `calypso_reader_following_intro_render`. Thanks for asking!